pkg/apis/mayadata.io/v1alpha1/backup_schedule.go

```diff
@@ -73,14 +91,18 @@ type PeriodicFullBackupConfig struct {
 type DMaaSBackupPhase string
 
 const (
+	// DMaaSBackupPhaseActive represents the active phase of DMaaSBackup
+	DMaaSBackupPhaseActive DMaaSBackupPhase = "Active"
```

Comment:
this is not used. We can think of removing it in next PR or release. Not an urgent one.
Instead of this, 'InProgress' is being used if I'm not wrong.

Reply:
As of now we are not updating spec.state -- it will remain empty only, we are updating status.phase with inprogress. `Active` will be set during reconcilation and other annotation/label/finalizer also.